java/email-contact-form/src/Cors.java

```diff
@@ -0,0 +1,55 @@
+package io.openruntimes.java.src;
+
+import io.openruntimes.java.RuntimeContext;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.HashMap;
+import java.lang.*;
+
+
+public class Cors {
+
+    /*
+     * Returns true if the origin is allowed to make requests to this endpoint
+     *
+     * Parameters:
+     *   req: Request object
+     *
+     * Returns:
+     *   (boolean): True if the origin is allowed, False otherwise
+     */
+    public static boolean isOriginPermitted(RuntimeContext context) {
+        String allowedOrigins = System.getenv("ALLOWED_ORIGINS");
+        if (allowedOrigins == null || allowedOrigins.equals("*")) {
+            return true;
+        }
+
+        List<String> allowedOriginsList = Arrays.asList(allowedOrigins.split(","));
+        String originHeader = context.getReq().getHeaders().get("Origin");
```

Comment:
Can we access with lowercase origin?

String originHeader = context.getReq().getHeaders().get("origin");

Reply:
Accessed the Origin headers with lowercase "origin"